doc/en/releasing.md

```diff
@@ -0,0 +1,12 @@
+# Releasing a new verison of the upgrader
+
+This document is aimed for internal consumption. To release a new version of the upgrader follow these steps.
+
+1. On your local system, clone the upgrader codebase. You need to work of the `master` branch from this point.
+2. Increment the version number in `src/upgrade-code`, commit your changes and push them back up to GitHub.
+3. Call `php build.php` on the command line. This will build the upgrader as a PHAR file. 
+4. Move `upgrade-code.phar` to a temporary folder.
+5. Switch to the `gh-pages` branch. e.g.: `git checkout gh-pages`
```

Comment:
Why are we pushing into the gh-pages branch? that's reserved for running a static website off the repo, if we upload it to the release page this step is not needed

Reply:
It's mimicking the set up that sspak is using. https://silverstripe.github.io/sspak/

Reply:
Yeah - I noticed that after reading the framework PR; I'm not sure that's the "right" approach, but we can live with it.